src/index.html

```diff
@@ -1,12 +1,413 @@
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__content">
+    <header class="page__header">
+      <a href="#home" class="page__header_logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__ul">
+          <li class="nav__list">
+            <a href="#Apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a href="#Samsung" class="nav__link">
+            Samsung
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a href="#Smartphones" class="nav__link">
+            Smartphones
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a
+              href="#Laptops & Computers"
+              class="nav__link"
+              data-qa="nav-hover"
+
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a href="#Gadgets" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a href="#Tablets" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a href="#Photo" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__list">
+            <a href="#Video" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page__catalog">
+      <div class="page__card page__card--1" data-qa="card">
+        <img
+        class="page__card_photo"
+        src="./images/imac.jpeg"
+        alt="card-imac"
+      >
+
+        <h2 class="page__card_name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="page__card_code">
+          Product code: 195434
+        </p>
+
+        <div class="page__card_rank">
+          <div class="page__stars">
+            <div class="page__stars_star"></div>
+            <div class="page__stars_star"></div>
+            <div class="page__stars_star"></div>
+            <div class="page__stars_star"></div>
+            <div class="page__stars_star"></div>
+          </div>
+          <p class="page__card_reviews">
+            Reviews: 5
+          </p>
+        </div>
+
+        <div class="page__card_cost">
+          <p class="page__card_price">
+            Price:
+          </p>
+          <h1 class="page__card_price-amount">
+            $2,199
+          </h1>
+        </div>
+        <button class="page__card_price-button" data-qa="card-hover">
+          BUY
+        </button>
+        </a>
+      </div>
+
+      <div class="page__card page__card--2">
+        <img
+        class="page__card_photo"
+        src="./images/imac.jpeg"
+        alt="card-imac"
+      >
```

Comment:
```suggestion
        <img
          class="page__card_photo"
          src="./images/imac.jpeg"
          alt="card-imac"
        >
```
Please fix code style according to https://github.com/mate-academy/layout_catalog/blob/master/checklist.md
Fix it not only here but everywhere